### PR TITLE
Improve lastIndexOf method

### LINKED ISSUE #342: Fix lastIndexOf method
There is currently a method called `lastIndexOf` that lives in `/lib/lastIndexOf.js`.
It is incomplete and needs to be fixed!

### Acceptance Criteria:
1. Running `yarn test lastIndexOf` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by mtso**
Closes #342 

- [x] Running `yarn lint` does not trigger any linter errors
- [x] The test of the method you have fixed is passing
- [x] You have written a new skeleton method for someone else to work on!
- [x] You have written tests to accompany your skeleton method

### CODE REVIEW

File: lib/lastIndexOf.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+const isString = require("./isString");
+const isArray = require("./isArray");
```
Comment (ktilcu) [inline lib/lastIndexOf.js]: Thanks for dog fooding!

**Comment by ktilcu** [comment]
@mtso Thanks so much for your contributions! Could you open an issue for the new skeleton method you created? It would be really helpful (i'm gonna copy/paste this to your other PR's too...heh :neckbeard:

**Comment by ktilcu** [comment]
samesies
